### ISSUE TITLE
NDCube 2.1.1 release broke Specutils docs build

### ISSUE THREAD

**Issue by pllim** [sunpy/ndcube#604]
### Describe the bug

Follow up of #598 to see if #600 fixed the problem. Now see other warnings.

Example log: https://readthedocs.org/projects/specutils/builds/19743252/

```
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.crop:35: WARNING: py:obj reference target not found: ndcube..ndcube.NDCubeABC
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.crop_by_values:1: WARNING: py:obj reference target not found: ndcube.NDCube.crop()
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.crop_by_values:39: WARNING: py:obj reference target not found: ndcube.ndcube.NDCubeABC
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCube.rebin:28: WARNING: py:obj reference target not found: ndcube.utils.cube.propagate_rebin_uncertainty
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCube.rebin:99: WARNING: py:obj reference target not found: ndcube.NDCube.rebin
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:7: WARNING: py:obj reference target not found: 'interpolation'
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:7: WARNING: py:obj reference target not found: reproject.reproject_interp
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:7: WARNING: py:obj reference target not found: 'adaptive'
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:7: WARNING: py:obj reference target not found: reproject.reproject_adaptive
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:7: WARNING: py:obj reference target not found: 'exact'
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:7: WARNING: py:obj reference target not found: reproject.reproject_exact
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:19: WARNING: py:obj reference target not found: True`
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:46: WARNING: py:obj reference target not found: reproject.reproject_interp
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:48: WARNING: py:obj reference target not found: reproject.reproject_adaptive
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:50: WARNING: py:obj reference target not found: reproject.reproject_exact
ndcube/ndcube.py:docstring of ndcube.ndcube.NDCube.to:25: WARNING: py:obj reference target not found: ǸDCube
```

### To Reproduce

Run specutils doc build with ndcube 2.1.1 and turn warnings into errors.

### Screenshots

N/A

### System Details

See RTD log above.

### Installation method

pip

**Comment by pllim** [comment]
Okay, I am not going crazy... these are real formatting errors.

https://github.com/sunpy/ndcube/blob/6425f7c4ba6054109ac2a793d4a7acd4f29acb42/ndcube/ndcube.py#L215

**Comment by nabobalis** [comment]
We currently do not run nitpicky on ndcube. I can try to patch these sometime this weekend.

**Comment by nabobalis** [comment]
So while I have #607 in draft and with nitpicky on and fixed as far as I can tell.

Specutils fails with:
```                                                                                                                                                                                                                                                                         
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.axis_world_coords_values:1: WARNING: py:meth reference target not found: ndcube.NDCube.axis_world_coords
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.crop:35: WARNING: py:obj reference target not found: ndcube.NDCubeABC
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.crop_by_values:39: WARNING: py:obj reference target not found: ndcube.NDCubeABC
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCube.rebin:28: WARNING: py:obj reference target not found: ndcube.utils.cube.propagate_rebin_uncertainties
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCube.rebin:99: WARNING: py:meth reference target not found: ndcube.NDCube.rebin
```

These are strange as they currently all link in my draft PR. Clearly I am missing something. 

and
```
/home/nabil/Git/ndcube/ndcube/visualization/mpl_plotter.py:docstring of ndcube.visualization.mpl_plotter.MatplotlibPlotter.plot:30: WARNING: py:obj reference target not found: mpl_animators.ArrayAnimatorWCS
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:7: WARNING: py:obj reference target not found: reproject.reproject_interp
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:7: WARNING: py:obj reference target not found: reproject.reproject_adaptive
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:7: WARNING: py:obj reference target not found: reproject.reproject_exact
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:46: WARNING: py:obj reference target not found: reproject.reproject_interp
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:48: WARNING: py:obj reference target not found: reproject.reproject_adaptive
/home/nabil/Git/ndcube/ndcube/ndcube.py:docstring of ndcube.ndcube.NDCubeBase.reproject_to:50: WARNING: py:obj reference target not found: reproject.reproject_exact
```

The reproject issues need `reproject` and ``mpl_animators`` adding to the `intersphinx_mapping` as far as I can tell. You can add them to the nitpick ignore file instead.

As a result of this, I opened https://github.com/astropy/specutils/pull/1041 to hopefully "fix" any issues on that side.

**Comment by DanRyanIrish** [comment]
Has this been resolved?

**Comment by nabobalis** [comment]
> Has this been resolved?

If you review and merge my PR.

**Comment by DanRyanIrish** [comment]
I'm on it!

**Comment by nabobalis** [comment]
I am still waiting for #607 to be reviewed and merged. Then I will re-update my PR on specutils.

Sorry for the delay.

**Comment by nabobalis** [comment]
#607 was merged and I tagged a 2.1.2 release. 

I am going to close this and hope that the tracking PR on specutils is enough.

**Comment by pllim** [comment]
Thank you!
